app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<% provide :page_title, construct_page_title(t('.title')) %>
```

Comment:
Let's set these page titles to something better then just 'ERA' like we do in HydraNorth. Using the sufia construct_page_title but we may want to over ride this as it generates `Login // Sufia` in this case where it should be maybe using a dash instead like so `Login - Sufia`. Sufia is being used as this is the current site name as of right now.